### PR TITLE
Feature scaffold luminaries

### PR DESCRIPTION
**Pull request by varffs**
- adds post type
- scaffold index and single
- add base of related posts partial
- add sorting filters for archive
- needs full styling and image size handling (in future)

### CODE REVIEW

File: archive-luminaries.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      </div>
+    </div>
+    <div id="posts" class="grid-row">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+?>
+        <article <?php post_class('grid-item item-s-6 item-m-3 margin-bottom-small'); ?> id="post-<?php the_ID(); ?>" data-sort-order="<?php echo $post->menu_order; ?>" data-sort-alphabetical="<?php echo $post->post_name; ?>">
```
Comment (mikeyrayvon) [inline archive-luminaries.php]: i suggest `item-m-4 item-l-3` here. what do think of that?

File: js/main.js [hunk 2]
```diff
@@ -70,4 +73,58 @@ Site.News = {
...
+      $(this).hide();
+      $('#luminaries-sort-alphabetical').show();
+    });
+  },
+
+  sort: function(type) {
+    var $posts = $('#posts');
+
+    $posts.find('.type-luminaries').sort(function(a, b) {
+      if (type === 'order') {
```
Comment (mikeyrayvon) [inline js/main.js]: could u add some more comments on this stuff?  im lost

File: archive-luminaries.php (hunk 1 above)
Comment (varffs) [inline archive-luminaries.php]: Yeah I think I agree. Lets do that in detailing

File: js/main.js (hunk 2 above)
Comment (varffs) [inline js/main.js]: I'll do it on master 🎱

**Pull request merged**